justfile

```diff
@@ -14,8 +14,6 @@ default: mise mvn
 
 # mise install
 mise:
-    mise plugin install --quiet maven
-    mise plugin install --quiet mvnd https://github.com/joschi/asdf-mvnd
```

Comment:
Non-core plugin configuration can move [into `mise.toml`](https://github.com/liftwizard/klass/blob/main/mise.toml) at the downside of not being able to use .tool-versions.